src/actions/transitionProperties.ts

```diff
@@ -68,8 +68,8 @@ export const showTransitionProperties = (component: string, stateMachine: string
                         messageType,
                         jsonMessageString,
                         id: firstId,
-                        privateTopic: (privateTopic) ? privateTopic : session.getDefaultPrivateTopic(),
-                        privateTopics: session.getPrivateTopics()
+                        privateTopic: (privateTopic) ? privateTopic : session.privateTopics.getdefaultPublisherTopic(),
```

Comment:
get*D*efaultPublisherTopic

Reply:
The problem is in the API

Reply:
Ok. created a ticket for that:

https://github.com/xcomponent/ReactiveXComponent.py/issues/32

Reply:
Wrong API, you created it in the python API